arthur/apis/cloudflare/zones.py

```diff
@@ -0,0 +1,37 @@
+"""APIs for managing Cloudflare zones."""
+from typing import Optional
+
+import aiohttp
+
+from arthur.config import CONFIG
+
+CF_TOKEN = CONFIG.cloudflare_token
+
+
+async def list_zones(zone_name: Optional[str] = None) -> dict[str, str]:
+    """List all Cloudflare zones."""
+    endpoint = "https://api.cloudflare.com/client/v4/zones"
+    request_headers = {"X-Auth-User-Service-Key": CF_TOKEN}
+
+    if zone_name is not None:
+        endpoint += f"?name={zone_name}"
+
+    async with aiohttp.ClientSession() as session:
+        async with session.get(endpoint, headers=request_headers) as response:
+            info = await response.json()
+
+    zones = info["result"]
+
+    return {zone.name: zone.id for zone in zones}
```

Comment:
These aren't actual attributes, it's a dictionary, it must be accessed with indexes instead.